modality-probe-cli/src/export.rs

```diff
@@ -5,24 +5,25 @@ use std::{collections::HashMap, path::PathBuf, str::FromStr};
 
 use structopt::StructOpt;
 
-use crate::graph::{self, digraph::Digraph, meta::*, Cfg};
+use crate::{
+    component::Component,
+    graph::{self, digraph::Digraph, meta::*, Cfg},
+};
 
 /// Export a textual representation of a causal graph using the
 /// collected coumnar form as input.
 #[derive(Debug, PartialEq, StructOpt)]
 pub struct Export {
     /// Generate the graph showing only the causal relationships,
     /// eliding the events inbetween.
-    #[structopt(long)]
+    #[structopt(short, long)]
     pub interactions_only: bool,
-    /// The path the probes.csv for a component.
-    #[structopt(long)]
-    pub probes: PathBuf,
+    /// The path to a component directory. To include multiple
+    /// components, provide this switch multiple times.
+    #[structopt(short, long)]
```

Comment:
Maybe worth adding `required = true` to have the cli complain about missing `--components`?

Reply:
Aye. https://github.com/auxoncorp/modality-probe/pull/171/commits/21a569afdff6c44ef0ef9a724cc197a310e5093a